internal/daemon/hostsidemanager.go

```diff
@@ -94,6 +94,14 @@ func NewHostSideManager(vsp plugin.VendorPlugin, opts ...func(*HostSideManager))
 		opt(h)
 	}
 
+	addr, port, err := h.vsp.Start()
+	if err != nil {
+		h.log.Error(err, "VSP Start() returned error")
+		return nil
+	}
+	h.addr = addr
+	h.port = port
```

Comment:
moved here, why?

Reply:
Also, error handling is not correct anymore.

Reply:
Because SetNumVf API is called before start if not moved here.